books-call-number/nodes/zipReport.json

```
@@ -0,0 +1,22 @@
+{
+  "id": "6b08fa16-772e-4587-81db-5121f1489990",
+  "name": "Compress book report",
```

```suggestion
  "name": "Compress Book Report",
```
Not required, but I suggest being grammatically correct.
As these are labels, aka Titles, the first letter of each word should be capitalized.

---

Resolved